app/screens/more_dms/more_dms.js

```diff
@@ -471,13 +476,13 @@ export default class MoreDirectMessages extends PureComponent {
                         keyboardAppearance={getKeyboardAppearanceFromTheme(theme)}
                         value={term}
                     />
-                    <SelectedUsers
-                        selectedIds={this.state.selectedIds}
-                        warnCount={5}
-                        maxCount={7}
-                        onRemove={this.handleRemoveProfile}
-                    />
                 </View>
+                <SelectedUsers
```

Comment:
this was changed with the SearchBar PR, but the chips were placed on top of the searchbar, this fixes it